app/views/ad_templates/sponsored-text/themes/dark.css

```diff
@@ -0,0 +1,22 @@
+#cf {
+  text-decoration: none;
+  color: #aaaaaa;
+  line-height: 1.7;
+  font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
```

Comment:
Does unquoted `Helvetica Neue` work correctly? I thought we deleted that entry from the other stylesheets.

Reply:
According to the browser, it works